spec/ttml1.xml

```diff
@@ -4598,12 +4598,19 @@ as a style property only to those element types indicated in the following table
 computed value of the style property must be considered to be no less than the
 size of the largest font size that applies to the element and its
 descendant elements in the intermediate synchronic document as determined by
-<specref ref="semantics-region-layout-step-1"/>. However, the value <code>normal</code> is not relative,
-and is therefore inherited as-is, as specified at <specref ref="semantics-style-resolution-processing-css"/>.</p>
+<specref ref="semantics-region-layout-step-1"/>. Furthermore,
+when performing <specref ref="semantics-style-resolution-processing-css"/>, step (4), [relative value resolution],
+the value <code>normal</code> is considered to be <emph>relative</emph> only on the most nested elements to which this property applies, namely,
+on <el>p</el> elements; otherwise, it is not considered to relative.</p>
 <note role="elaboration">
```

Comment:
Why would the value `normal` ever be considered relative?

Reply:
When it is computed to be 125% of the font size of the paragraph; ergo, it is relative to the computed value of the font size of the paragraph.

Reply:
In TTML1, it is not 125% of the font size of the paragraph, it _must be considered to be no less than the size of the largest font size that applies to the element and its descendant elements in the intermediate synchronic document as determined by 9.3.2 Intermediate Synchronic Document Construction._

Reply:
more precisely, it is ``min(``*what you say*,*whatever the implementation wants but is recommended to be 125% font size*``)``, but, in any case, that still makes it relative on p, which is what you asked for

Reply:
I disagree @skynavga , I think the specified value `"normal"` is _never_ relative, but does generate a computed line height that is indeed related to the font size.

Reply:
@nigelmegitt then your understanding is at variance with both (1) XSL-FO, which, e.g., considers ``auto`` (on any property) to be relative, and (2) the computed style set processing algorithm, which requires a value to be treated as relative in order to map its category to ``computed``

from XSL-FO 1.1 &#x00A7;5.1.2, we have (emphasis added)

>Specified values may be absolute (i.e., they are not specified relative to another value, as in "red" or "2mm") or relative (i.e., they are specified relative to another value, as in **"auto"**, "2em", and "12%"), or they may be expressions.

Reply:
@skynavga would you not allow that `"normal"` is an expression then, and neither absolute nor relative?

Reply:
@nigelmegitt in this context, *expression* is referring to the XSL-FO expression language defined in https://www.w3.org/TR/xsl11/#d0e5032; so, while 'normal' could potentially be interpreted as an EnumerationToken (in the XSL-FO expression language), that interpretation doesn't really apply in TTML;

the bottom line is that in TTML a style value can only be designated as a computed value in a computed style set when it is a relative value; otherwise, it is a specified value in a computed style set;

furthermore, the language I cited today from TTML1 &#x00A7;8.2.12, namely

>If the value of this attribute is normal, then the computed value of the style property must be ...

makes sense only when it is assigned the category ``computed``, and that can only happen if it is relative (per &#x00A7;8.4.4.3, step 4);

Reply:
@skynavga okay, thanks for the explanation. I may be repeating part of the conversation on Thursday, but it seems like we need to be able to inherit "normal" as a specified value on elements where it does not _apply_ and only compute the value on elements where it does apply.

Reply:
yes, that is exactly what I've been trying to say

Reply:
OK, this works for me, modulo the two comments I just added about specific bits of the wording.